sdk/cosmos/azure-cosmos-tests/pom.xml

```diff
@@ -0,0 +1,451 @@
+      --add-opens com.azure.cosmos/com.azure.cosmos.rx.changefeed.epkversion=ALL-UNNAMED
+
+      --add-opens com.azure.cosmos/com.azure.cosmos.implementation=ALL-UNNAMED
+      --add-opens com.azure.cosmos/com.azure.cosmos.implementation.batch=ALL-UNNAMED
+      --add-opens com.azure.cosmos/com.azure.cosmos.implementation.batch=com.fasterxml.jackson.databind
+      --add-opens com.azure.cosmos/com.azure.cosmos.implementation.caches=ALL-UNNAMED
+      --add-opens com.azure.cosmos/com.azure.cosmos.implementation.cpu=ALL-UNNAMED
+      --add-opens com.azure.cosmos/com.azure.cosmos.implementation.clienttelemetry=ALL-UNNAMED
+      --add-opens com.azure.cosmos/com.azure.cosmos.implementation.directconnectivity=ALL-UNNAMED
+      --add-opens com.azure.cosmos/com.azure.cosmos.implementation.directconnectivity.rntbd=ALL-UNNAMED
+      --add-opens com.azure.cosmos/com.azure.cosmos.implementation.directconnectivity.TcpServerMock=ALL-UNNAMED
+      --add-opens com.azure.cosmos/com.azure.cosmos.implementation.directconnectivity.TcpServerMock.rntbd=ALL-UNNAMED
+      --add-opens com.azure.cosmos/com.azure.cosmos.implementation.directconnectivity.TcpServerMock.rntbd=com.fasterxml.jackson.databind
+      --add-opens com.azure.cosmos/com.azure.cosmos.implementation.feedranges=ALL-UNNAMED
+      --add-opens com.azure.cosmos/com.azure.cosmos.implementation.apachecommons.lang=ALL-UNNAMED
+      --add-opens com.azure.cosmos/com.azure.cosmos.implementation.guava25.base=ALL-UNNAMED
+      --add-opens com.azure.cosmos/com.azure.cosmos.implementation.http=ALL-UNNAMED
+      --add-opens com.azure.cosmos/com.azure.cosmos.implementation.patch=ALL-UNNAMED
+      --add-opens com.azure.cosmos/com.azure.cosmos.implementation.query=ALL-UNNAMED
+      --add-opens com.azure.cosmos/com.azure.cosmos.implementation.routing=ALL-UNNAMED
+      --add-opens com.azure.cosmos/com.azure.cosmos.implementation.throughputControl=ALL-UNNAMED
+      --add-opens com.azure.cosmos/com.azure.cosmos.implementation.throughputControl.controller=ALL-UNNAMED
+      --add-opens com.azure.cosmos/com.azure.cosmos.implementation.throughputControl.controller.request=ALL-UNNAMED
+      --add-opens com.azure.cosmos/com.azure.cosmos.implementation.changefeed=ALL-UNNAMED
+      --add-opens com.azure.cosmos/com.azure.cosmos.implementation.changefeed.pkversion=ALL-UNNAMED
+      --add-opens com.azure.cosmos/com.azure.cosmos.implementation.changefeed.epkversion=ALL-UNNAMED
+      --add-opens com.azure.cosmos/com.azure.cosmos.implementation.faultinjection=ALL-UNNAMED
+    </javaModulesSurefireArgLine>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-cosmos</artifactId>
+      <version>4.43.0-beta.1</version> <!-- {x-version-update;com.azure:azure-cosmos;current} -->
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-cosmos-test</artifactId>
```

Comment:
azure-cosmos-test will bring in azure-cosmos, do we need azure-cosmos dependency?

Reply:
Doesn't hurt - and if we want to change it I can do it after merging this PR.